### PR TITLE
cleaning up

### PR DESCRIPTION
**Pull request by HailSanta**
refactor some hardcoded area/map IDs
remove some padding and unused variables
remove all map RODATA padding
fix SetUP_* event script templates

### CODE REVIEW

File: src/7BB60.c [hunk 1]
```diff
@@ -836,31 +836,16 @@ void collision_check_player_overlaps(void) {
...
-                case 1:
-                    if (playerStatus->pos.x >= -300.0f && playerStatus->pos.x <= -140.0f) {
-                        ret = FALSE;
-                    }
-                    break;
-            }
-        }
+    if (PlayerSlidingCallback != NULL) {
+        return PlayerSlidingCallback();
     }
```
[the remark this answers is not in the document]
Comment (HailSanta) [reply src/7BB60.c:844]: Good catch!

File: src/world/area_flo/flo_11/flo_11_2_main.c [hunk 2]
```diff
@@ -24,18 +24,18 @@ EvtScript N(EVS_ExitWalk_flo_12_0) = EVT_EXIT_WALK(60, flo_11_ENTRY_1, "flo_12",
 }
 
 // template for binding the event for the generic EVS_OnEnterPipe
-#define SetUP_PIPE(entry, collider, exitScript) \
+#define EVT_SETUP_PIPE(entry, collider, exitScript) \
```
[the remark this answers is not in the document]
Comment (HailSanta) [reply src/world/area_flo/flo_11/flo_11_2_main.c:27]: Quite a few things to backport in this one. Should I also fix the `SetUP_CAMERA_*` in this PR?

[the remark this answers is not in the document]
Comment (bates64) [reply src/world/area_flo/flo_11/flo_11_2_main.c:27]: Please

**Review by bates64 — APPROVED** [review]
phys_set_landing_adjust_cam_check - nice!

**Pull request merged**